src/clip.sol

```diff
@@ -248,7 +251,7 @@ contract Clipper {
         sales[id].tic = uint96(block.timestamp);
 
         uint256 top;
-        top = rmul(getPrice(), buf);
+        top = max(rmul(getPrice(), buf), tab / lot);
```

Comment:
Still thinking through this. I can see Lucas's point, but I see this as an emergency fallback, the oracle price will still be primarily used as the starting point unless there's a hugely dramatic drop.

Primoz has a point, too, but rather than reducing it by a multiplier I'd argue for Lucas' point that this is just a starting point for the auction and that at least gives the vaultholder a fair shot at recapitalizing in the market. If indeed it is too low, it should still be repriced in the `redo()`